### PR TITLE
Cms/jimklimov/stories/user story/jenkins is the way for networkupstools/index

### PR DESCRIPTION
**Pull request by jimklimov**
Posting a success story for NUT multi-platform builds

### CODE REVIEW

File: src/user-story/jenkins-is-the-way-for-networkupstools/index.yaml [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+      scenarios run on CircleCI and Appveyor to take advantage of some free FOSS
+      builds on MacOS and Windows respectively.
+    - Of notable interactions with the Jenkins community, this and related
+      efforts led to improvements in the git-client-plugin to extend the use of
+      reference repositories, instant-messaging-plugin and ircbot-plugin for
+      querying Jenkins server status via IRC servers, lockable-resources plugin
+      for many itches scratched here and there, and pipeline-groovy-lib-plugin
+      for easier use of test library versions along with test pipeline branches.
+      And very many lessons learned and friends made :)
+---
```
Comment (halkeye) [inline src/user-story/jenkins-is-the-way-for-networkupstools/index.yaml:124]:
```suggestion
```

You have multiple yaml documents in this file (--- is a divider of documents), you need to remove the second one.

Comment (jimklimov) [reply src/user-story/jenkins-is-the-way-for-networkupstools/index.yaml]: Well, that's what CMS generated and submitted... Thanks!

**Comment by halkeye** [comment]
@alyssat @MarkEWaite https://deploy-preview-14--jenkins-is-the-way.netlify.app/user-story/jenkins-is-the-way-for-networkupstools

**Review by MarkEWaite — APPROVED** [review]
That is a great story!  Heartily approve!

Thanks @jimklimov !

**Pull request merged**